### ISSUE TITLE
Please share the training recipe.

### ISSUE THREAD

**Issue by luowyang** [alibaba-mmai-research/HyRSM#4]
The training recipe (GPU model, how many query videos per task, how many tasks per batch, how many iterations/tasks/batches in total, optimizer options, learning rate policy, etc.) is not available in the paper, nor is it clear in the code (the options in the YAML are mysterious, some options seem redundant, some hyperparameters are embedded in the code, etc.). It is not possible to reproduce the results from scratch without the recipe. Please, summarize all the hyperparameters, policies, and settings that have a place in the training procedure.
